src/flagStore.ts

```diff
@@ -60,7 +60,11 @@ export class FlagStore {
 		}
 
 		try {
-			const flags = await this.api.getFeatureFlags(this.config.project, this.config.env);
+			const getFlags = await this.api.getFeatureFlags(this.config.project, this.config.env);
+			const flags = getFlags.map(flag => {
+				flag.variationLength = flag.variations.length;
```

Comment:
🤔 Any reason why we can't always reference `flag.variations.length` instead of copying the length to a new field?

Reply:
yeah I think that should be doable I'll make the change and test it